src/kibali/ProtectedResource.cs

```diff
@@ -230,18 +230,32 @@ public string GeneratePermissionsTable(Dictionary<string, List<AcceptableClaim>>
         {
             var permissionsStub = new List<string> { "**TODO: Provide applicable permissions.**" };
```

Comment:
Should we still have this given that we have a permissions file as a source of truth? If any permissions need to be added they ought to be put in the permissions file.